src/plugins/interactive_setup/public/enrollment_token_form.tsx

```diff
@@ -226,3 +224,59 @@ export function compareAddresses(a: string, b: string) {
   }
   return 0;
 }
+
+export const EnrollmentTokenHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const button = (
+    <EuiLink onClick={() => setIsPopoverOpen((isOpen) => !isOpen)}>
+      <FormattedMessage
+        id="interactiveSetup.enrollmentTokenHelpPopover.buttonText"
+        defaultMessage="Where do I find this?"
+      />
+    </EuiLink>
+  );
+
+  return (
+    <EuiPopover
+      button={button}
+      anchorPosition="rightCenter"
+      isOpen={isPopoverOpen}
+      closePopover={() => setIsPopoverOpen(false)}
+    >
+      <EuiText size="s" grow={false}>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.whenHelpText"
+            defaultMessage="The enrollment token is automatically generated when you start Elasticsearch for the first
+          time."
+          />
+        </p>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.whereHelpText"
+            defaultMessage="You might need to scroll back a bit in the terminal to view the enrollment token."
+          />
+        </p>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.howHelpText"
+            defaultMessage="To generate a new enrollment token run the following command in the Elasticsearch
+          directory:"
+          />
+        </p>
+        <EuiCodeBlock language="bash" paddingSize="m" isCopyable>
+          bin/elasticsearch-create-enrollment-token --scope kibana
+        </EuiCodeBlock>
+      </EuiText>
+      <EuiPopoverFooter>
+        <DocLink app="elasticsearch" doc="configuring-stack-security.html">
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.docLinkText"
+            defaultMessage="Learn how to setup Elastic."
```

Comment:
```suggestion
            defaultMessage="Learn how to set up Elastic."
```

Reply:
I'm curious about whether to use a period here. We use punctuation in other places where it's necessary, such as **Forgot password?**, but the period seems unnecessary here. @gchaps or @KOTungseth -- what's the convention here?

Reply:
We've been using punctuations for links to documentation previously:

<img width="886" alt="Screenshot 2021-11-22 at 18 04 47" src="https://user-images.githubusercontent.com/190132/142912737-e852c5ab-c2ed-441a-90f8-0ffe4e42dd67.png">

Not saying this is the convention but just FYI :)

Reply:
Thanks for the explanation @thomheymann. I wasn't sure what the common convention is.